book/networking/p2p/network/README.md

```diff
@@ -0,0 +1,69 @@
+# Network
+
+TODO: Some high-level overview of peer-to-peer networking, either here or in a short chapter one level up in the tree for `p2p`
+
+The `network` crate is responsible for managing the node's connection to the Ethereum peer-to-peer (P2P) network, enabling communication with other nodes via the [various P2P subprotocols](https://github.com/ethereum/devp2p).
+
+TODO: Link to the "bird's eye docs" in `crates/net/network/src/lib.rs`?
+
+Reth's P2P networking consists primarily of 4 ongoing tasks:
+- **Discovery**: Discovers new peers in the network
+- **Transactions**: Accepts, requests, and broadcasts mempool transactions
+- **ETH Requests**: Responds to incoming requests for headers and bodies
+    - TODO: Doesn't also issue requests for these?
```

Comment:
outgoing requests are handled elsewhere: `StateFetcher`

Reply:
will note this down in the chapter, but i'm inclined to ask why - as someone approaching this from 1000ft up and digging down, it feels like a clean abstraction to be able to say that the network management task (`NetworkManager`) simply routes messages to the appropriate other tasks & manages state of peers, while the other tasks (e.g. `EthRequestHandler`) manage both incoming & outgoing messages related to that task